angular/src/lib/tooltip/tooltip-container.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component, Input,  TemplateRef, ElementRef} from '@angular/core';
+
+
+
+@Component({
+  selector: 'md-tooltip-container',
+  styleUrls: ['tooltip-container.scss'],
+  template: `
+  <div class="md-tooltip md-event-overlay md-event-overlay--arrow">
```

Comment:
I am not sure if you can build the class="md-tooltip md-event-overlay md-event-overlay--arrow" to the host.  The host here seems to be a tag without any attribute.

Reply:
Yeah, add these classes to the host so it we can take out the div wrapper.

Reply:
refactored how I was adding these classes .. host didn't worked but found a better way